[feat] Removed model from deletion return and added is_deletion flag.…

bensontwu opened the pull request:
… Added cfn test inputs

Commit:
[feat] Removed model from deletion return and added is_deletion flag. Added cfn test inputs

jaoanan1126 reviewed (commented):
If possible for the contract tests let's make the input `BucketName` dynamic . If not let's leave it generic and I can create a readme instructing the user to change it.

Commit:
bugfix(namespace): f
ixed namespace

Commit:
[feat] added example template to deploy contract test dependencies and modified input to use export from example template. Added readme

Commit:
[fix] Fixed list handler keyerror

bensontwu commented:
I added a new example template to deploy an S3 Bucket that the contract tests can reference dynamically. I updated the inputs to reference this property and added a Readme. Also did a minor fix for List.

Commit:
Merge branch 'bugfix/namespace' into cfn-test-handler-updates

Commit:
[fix] Fixed discrepencies in update handler
